webapp/src/gitjson.tsx

```diff
@@ -778,12 +778,20 @@ class GithubComponent extends data.Component<GithubProps, GithubState> {
                         </div>
                     </div>
                     <div className="rightHeader">
+                        <sui.Link className="ui button neutral" icon="external alternate" href={url} title={lf("Open repository in GitHub.")} target="_blank" onKeyDown={fireClickOnEnter} />
+                        {!isBlocksMode &&
+                            isOwner &&
+                            <sui.Link
```

Comment:
i know you're just moving this code, but should this also have fire click on enter?